harvard-university-of-worcester.csl

```diff
@@ -261,14 +261,14 @@
       </else>
     </choose>
   </macro>
-  <citation collapse="year" disambiguate-add-year-suffix="true" et-al-use-first="3" et-al-min="4">
+  <citation et-al-min="3" et-al-use-first="1" sort-separator="," disambiguate-add-year-suffix="true" collapse="year" cite-group-delimiter=" ">
```

Comment:
you already did that by removing the comma in the delimiter in line 269 below. Just remove `cite-group-delimiter=" "`

@rmzelle -- I can't get cite-group-delimiter to do anything in my Zotero; could you check if you can?